hazelcast/src/main/java/com/hazelcast/datastore/JdbcDataStoreFactory.java

```diff
@@ -40,6 +41,7 @@
 @Beta
 public class JdbcDataStoreFactory implements ExternalDataStoreFactory<DataSource> {
 
+    private static final int JDBC_TEST_CONNECTION_TIMEOUT_SECONDS = 1;
```

Comment:
This seems to be quite low, I can easily imagine a situation where this times out, but the connection is ok (e.g. the database is in a different geographical region, and the network is a bit flaky). Let's increase it.

Reply:
Increased to 5